Peggy-Debugging.md

```diff
@@ -0,0 +1,31 @@
+# Current Peggy Issues
```

Comment:
Should this file be removed?  (not a blocker)

Reply:
Probably